test/splitting/annotations.unit

```diff
@@ -0,0 +1,9 @@
+40 columns                              |
+>>> blet
```

Comment:
"blet"? :)

Reply:
Whoops.  An old college friend of mine used to use that word in place of "foo" all the time and I kind of borrowed it from him.  In this case it means "Hey Paul, don't forget to replace this with a meaningful comment"... and clearly I did forget.